File: app/src/main/java/com/haker/ah/CloudFragment.java
package com.haker.ah;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.DefaultItemAnimator;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import java.util.ArrayList;

public class CloudFragment extends Fragment {
    private ArrayList<CloudData> itemList = new ArrayList<>();
    private RecyclerView recyclerView;
    private CloudAdapter mAdapter;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.fragment_cloud, container, false);
        // Inflate the layout for this fragment
        recyclerView = (RecyclerView) v.findViewById(R.id.recycler_view);
        recyclerView.setHasFixedSize(true);
        mAdapter = new CloudAdapter(itemList);

        RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(getActivity());
        recyclerView.setLayoutManager(mLayoutManager);
        recyclerView.setItemAnimator(new DefaultItemAnimator());
        recyclerView.setAdapter(mAdapter);
        return v;

    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        prepareData();
    }

    private void prepareData() {
        itemList.add(new CloudData("구르르름",37.54892296550104,126.99089033876304));
        itemList.add(new CloudData("힘드러",37.54892296550104,126.99089033876304));
        itemList.add(new CloudData("난 여기랑 안맞나바",37.54892296550104,126.99089033876304));
        itemList.add(new CloudData("개발이 머야",37.54892296550104,126.99089033876304));
        itemList.add(new CloudData("울고싶다",37.54892296550104,126.99089033876304));
    }
}

File: app/src/main/java/com/example/alt_map/Cloud_fragment.kt
package com.example.alt_map

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.TextView
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.DefaultItemAnimator
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.example.alt_map.R.id.rcv_weather


class Cloud_fragment : Fragment() {
    private var itemList: ArrayList<CloudData> = ArrayList()
    private lateinit var recyclerView: RecyclerView
    private lateinit var mAdapter: CloudAdapter

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val v = inflater.inflate(R.layout.fragment_cloud, container, false)
        recyclerView = v.findViewById(rcv_weather)
        recyclerView.setHasFixedSize(true)

        mAdapter = CloudAdapter(requireContext())

        recyclerView.layoutManager = LinearLayoutManager(requireContext(), LinearLayoutManager.VERTICAL, false)
        recyclerView.itemAnimator = DefaultItemAnimator();
        recyclerView.adapter = mAdapter

        addData()

        return v
    }
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

    }

    private fun addData() {
        mAdapter.data.add(CloudData(R.drawable.img,"축구", 37.54892296550104, 126.99089033876304))
        mAdapter.data.add(CloudData(R.drawable.img_1,"외출", 37.54892296550104, 126.99089033876304))
        mAdapter.data.add(CloudData(R.drawable.img_2,"실내코딩", 37.54892296550104, 126.99089033876304))
        mAdapter.data.add(CloudData(R.drawable.img_3,"농구", 37.54892296550104, 126.99089033876304))
        mAdapter.data.add(CloudData(R.drawable.img_4,"헬스", 37.54892296550104, 126.99089033876304))
    }
}//